src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -136,10 +149,51 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     this.fetchIstioObjectDetailsFromProps(this.props.match.params);
   };
 
-  fetchIstioObjectDetailsFromProps = (props: IstioConfigId) => {
-    const promiseConfigDetails = props.objectSubtype
+  newIstioObjectPromise = (props: IstioConfigId, validate: boolean) => {
+    return props.objectSubtype
       ? API.getIstioConfigDetailSubtype(props.namespace, props.objectType, props.objectSubtype, props.object)
-      : API.getIstioConfigDetail(props.namespace, props.objectType, props.object, true);
+      : API.getIstioConfigDetail(props.namespace, props.objectType, props.object, validate);
+  };
+
+  fetchIstioObjectDetailsFromProps = (props: IstioConfigId) => {
+    const promiseConfigDetails = this.newIstioObjectPromise(props, true);
+
+    window.clearInterval(this.timerId);
+    this.timerId = window.setInterval(() => {
+      const timerPromise = this.newIstioObjectPromise(props, false);
+      timerPromise
+        .then(resultConfigDetails => {
+          if (resultConfigDetails.data && this.state.originalIstioObjectDetails) {
+            const [changed, type, newResourceVersion] = compareResourceVersion(
+              this.state.originalIstioObjectDetails,
+              resultConfigDetails.data
+            );
+            if (changed) {
+              AlertUtils.addWarning(
+                'Istio object type [' +
+                  type +
+                  '] name [' +
+                  props.object +
+                  '] has been modified externally to Kiali. New ResourceVersion:  ' +
+                  newResourceVersion
+              );
+            }
+          }
+        })
+        .catch(error => {
+          this.setState({
+            isRemoved: true
+          });
+          AlertUtils.addError(
+            'Could not fetch Istio object type [' +
+              props.objectType +
```

Comment:
Should we use the name naming as in the alerts for the validations `type:name`?

![Screenshot of Kiali Console (23)](https://user-images.githubusercontent.com/613814/75774929-4899e200-5d51-11ea-8528-8edf32981183.png)

![Screenshot of Kiali Console (24)](https://user-images.githubusercontent.com/613814/75775018-6f581880-5d51-11ea-8a49-2d65af0fa21c.png)

Reply:
+1